examples/google-cloud-speech/output/google-cloud-speech/lib/google/cloud/speech/v1/cloud_speech_pb.rb

```diff
@@ -11,7 +11,7 @@
 require "google/protobuf/timestamp_pb"
 require "google/rpc/status_pb"
 
-# rubocop:disable Metrics/BlockLength, Metrics/LineLength
+# rubocop:disable Metrics/LineLength
```

Comment:
I think it would be safer to configure Rubocop to ignore `*_pb.rb` files. My understanding is that these files aren't in scope for this generator.

Reply:
@blowmage you're right the message types defined in these `_pb.rb`file is out of our control for now so that sounds like a good approach to me as well.

Reply:
OK, I'll update to ignore the *_pb.rb files.